### PR TITLE
chore: add idempotent insert in form schema after each deploy

### PR DESCRIPTION
**Pull request by Sepehr-Sobhani**
No card associated

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -135,19 +135,16 @@ deploy_db_migrations:
 	@$(SQITCH) --chdir mocks_schema deploy
 
 deploy_dev_data: ## deploy the database migrations with sqitch and load the data for local development
-deploy_dev_data: deploy_db_migrations
+deploy_dev_data: deploy_db_migrations deploy_prod_data
```
Comment (pbastia) [inline Makefile:138]: we'll want to guarantee the order here:
```suggestion
deploy_dev_data: | deploy_db_migrations deploy_prod_data
```

File: app/cypress/integration/cif/project-revision/index.spec.js [hunk 2]
```diff
@@ -252,6 +252,11 @@ describe("the new project page", () => {
     cy.findByRole("button", { name: /^submit/i }).click();
     cy.get(".error-detail").should("have.length", 4);
     cy.contains("Changes saved").should("be.visible");
+    // below assertion is not necessary, but it's here to make sure the happo
+    // snapshot is taken after the form is submitted
+    cy.findByText(/Add emissions intensity report/i)
+      .next()
+      .should("have.text", "Attention Required");
```
Comment (pbastia) [inline app/cypress/integration/cif/project-revision/index.spec.js:259]: nice.

File: schema/data/deploy-data.sh [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+__dirname="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+pushd "$__dirname" > /dev/null
+
+_psql() {
+  psql -d "$database" -h "$host" -p "$port" -U "$user" -qtA --set ON_ERROR_STOP=1 "$@" 2>&1
+}
+
+deployProdData() {
+  _psql -f "./prod/001_form.sql"
```
Comment (pbastia) [inline schema/data/deploy-data.sh:39]: Should we make this run all the files in `/prod/` ? Or do we prefer constraining which files go in there?

**Review by pbastia — CHANGES_REQUESTED** [review]

File: schema/data/deploy-data.sh (hunk 3 above)
Comment (Sepehr-Sobhani) [reply schema/data/deploy-data.sh:39]: I have seen this in ciip too but in there we are constraining which files to go. I'm not sure how many more insert SQL files we might need to add here but if it gets too much we might want to run all files within `/prod/` in a proper way. anyway, it's just my thoughts, what do you prefer?!

Comment (pbastia) [reply schema/data/deploy-data.sh:39]: We can leave this as is, and refactor when needed

**Review by pbastia — APPROVED** [review]

**Pull request merged**